frontend_lib/test/fixture/tracimLiveMessage/fileTlm.js

```diff
@@ -0,0 +1,11 @@
+import { author } from '../author.js'
+import { CONTENT_TYPE } from '../../../src'
```

Comment:
Is this import right? Maybe it's:
 ```suggestion
import { CONTENT_TYPE } from '../../../src/helper.js'
```